runtime/doc/treesitter.txt

```diff
@@ -676,16 +676,24 @@ add_directive({name}, {handler}, {force})                    *add_directive()*
 
     Parameters: ~
       • {name}     (string) Name of the directive, without leading #
-      • {handler}  function(match:string, pattern:string, bufnr:number,
-                   predicate:function, metadata:table)
+      • {handler}  function(match:table, pattern:string, bufnr:number,
+                   predicate:string[], metadata:table)
+                   • match(table): see |treesitter-query|
```

Comment:
Formatting here is a bit off, `match` looks like a function.

Go with
```lua
• match (table):
```

Or since the type is already mentioned on line 679, might be better to omit the type completely.

Reply:
done.